### PR TITLE
Feat: Realistic components; Add provisioning

### PR DESCRIPTION
**Pull request by joshuai96**
Add  components from SovereignCloudStack/issues#613.

### CODE REVIEW

File: kubernetes/environments/kind/api/provisioning.yaml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+  - displayname: Grafana
+    labels:
+      app: observability-cluster
+  - displayname: Prometheus
+    labels:
+      app: observability-cluster
+  - displayname: Zuul
+    labels:
+      system: gx-scs
+
```
Comment (okautz) [inline kubernetes/environments/kind/api/provisioning.yaml]: Nextcloud should be included
```suggestion
  - displayname: Nextcloud
    labels: {}
```

**Review by okautz — CHANGES_REQUESTED** [review]
Nextcloud should be added. See suggestion

File: kubernetes/environments/kind/api/provisioning.yaml (hunk 1 above)
Comment (joshmue) [reply kubernetes/environments/kind/api/provisioning.yaml]: IIRC, Nextcloud runs on an UCS installation, which also serves as IdP for Harbor.

So maybe also add UCS as IdP, here.

**Review by joshmue — APPROVED** [review]
As soon as the topic around Nextcloud/UCS is resolved: LGTM

**Comment by joshuai96** [comment]
@joshmue @o-otte I've added the UCS and labeled the Harbor and Nextcloud accordingly. See `kubernetes/environments/scs-public/api/provisioning.yaml`

**Review by okautz — APPROVED** [review]

**Pull request merged**